Model/Behavior/UploadBehavior.php

```diff
@@ -1344,7 +1344,7 @@ protected function _resizePhp(Model $model, $field, $path, $size, $geometry, $th
 				$outputHandler = 'imagepng';
 				$supportsQuality = true;
 				// convert 0 (lowest) - 100 (highest) thumbnailQuality, to 0 (highest) - 9 (lowest) quality (see http://php.net/manual/en/function.imagepng.php)
-				$adjustedQuality = intval((100 - $this->settings[$model->alias][$field]['thumbnailQuality']) / 100 * 9);
+				$adjustedQuality = (int)((100 - $this->settings[$model->alias][$field]['thumbnailQuality']) / 100 * 9);
```

Comment:
This change is OK for Merge, intval() old function PHP4 for format correct cast val in PHP 5. 
```php
$adjustedQuality = (int)((100 - $this->settings[$model->alias][$field]['thumbnailQuality']) / 100 * 9);
```